src/app/components/token-network-selector/token-network-selector.component.ts

```diff
@@ -48,6 +56,18 @@ export class TokenNetworkSelectorComponent implements ControlValueAccessor {
             ),
             share()
         );
+        this.ethBalance$ = this.raidenService.balance$;
+    }
+
+    ngOnInit() {
+        if (this.showServicesToken) {
+            this.tokens$ = combineLatest([
+                this.tokens$,
+                this.userDepositService.servicesToken$,
+            ]).pipe(
+                map(([tokens, servicesToken]) => [servicesToken, ...tokens])
```

Comment:
Keep in mind `servicesToken` **may** also be registered as a TokenNetwork, which may make it appear dupliicated if you do this kind of spread often

Reply:
Good point. I'll add a check. Thanks!